Chamasma/Common/Sources/Common/Extensions/Color+Extension.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Color+Extension.swift
+//
+//
+//  Created by Mariam Ormotsadze on 04.05.23.
+//
+import SwiftUI
+import UIKit
+
+public extension Color {
+    init?(hex: String) {
```

Comment:
You are doing something wrong. You should not have the same changes in all the PR's. 

One change should appear in one PR

Reply:
since that color extension was not merged, unfortunately, I needed that color extensions to make this component work fully, so, I know I should not do it, but I had to